crates/common/src/utils.rs

```diff
@@ -443,6 +443,22 @@ pub fn get_user_agent_with_version(req_headers: &HeaderMap) -> eyre::Result<Head
     Ok(HeaderValue::from_str(&format!("commit-boost/{HEADER_VERSION_VALUE} {}", ua))?)
 }
 
+// Trait for converting from u64 to B256
+pub trait FromU64 {
+    /// Converts a u64 to a B256 by left-padding with zeros
+    fn from_u64(value: u64) -> B256;
+}
+
+impl FromU64 for B256 {
+    /// Converts a u64 into a B256 by left-padding with zeros
+    fn from_u64(value: u64) -> B256 {
+        let mut buffer = [0u8; 32];
+        let bytes = value.to_be_bytes();
+        buffer[24..].copy_from_slice(&bytes);
+        B256::from(buffer)
+    }
+}
+
```

Comment:
Do we really need this? A conversion can be done by simply `B256::from(U256::from(u64_value))`

Reply:
I'm maintaining 2 branches in parallel - one with this and one without this, and that snuck in from the branch without it. I'll remove it from this one, thanks for the suggestion.